resources/camino-trekker/common/api.service.js

```diff
@@ -24,18 +24,18 @@ export const toursService = {
   },
 };
 
-function replaceObjectKeys([oldKeyName, newKeyName], obj) {
-  const ObjAsString = JSON.stringify(obj);
-  const newObjAsString = ObjAsString.replace(
-    new RegExp(`"${oldKeyName}":`, "g"),
-    `"${newKeyName}":`
-  );
-  return JSON.parse(newObjAsString);
-}
+// function replaceObjectKeys([oldKeyName, newKeyName], obj) {
```

Comment:
is this dead code or are we likely to bring it back?

Reply:
Good call. Probably dead. I originally wrote it to handle locale conversion ('English' to 'en') in trekker. But, after we talked last week, I agree that a normal migration is a better way to handle this in the long term. And type enums are a better temp fix.